Test/WebTestCase.php

```diff
@@ -134,30 +141,76 @@ protected function runCommand($name, array $params = array(), $reuseKernel = fal
     /**
      * Retrieves the output verbosity level.
      *
-     * @see Symfony\Component\Console\Output\StreamOutput for available levels
+     * @see Symfony\Component\Console\Output\OutputInterface for available levels
      *
-     * @return string
+     * @return int
+     *
+     * @throws \OutOfBoundsException If the set value isn't accepted
      */
-    private function retrieveVerbosityLevel()
+    protected function getVerbosityLevel()
     {
-        // Returns the local verbosity level
-        if ($this->verbosityLevel) {
-            $verbosity = 'StreamOutput::VERBOSITY_'.strtoupper($this->verbosityLevel);
-            if (defined($verbosity)) {
-                return $verbosity;
-            }
+        // If `null`, is not yet set
+        if (null === $this->verbosityLevel) {
+            // Set the global verbosity level that is set as NORMAL by the TreeBuilder in Configuration
+            $level = strtoupper($this->getContainer()->getParameter('liip_functional_test.command_verbosity'));
+            $verbosity = '\Symfony\Component\Console\Output\StreamOutput::VERBOSITY_'.$level;
```

Comment:
No, here it isn't possible to use the `use` statement as the `constant()` function cannot "intercept" it. The full namespace is needed to make the code work.